src/hash.rs

```diff
@@ -0,0 +1,85 @@
+use rustls::crypto::hash;
+use rust_symcrypt::hash::{Sha256State, Sha384State, HashState, sha256, sha384};

```

Comment:
>HashState [](http://example.com/codeflow?start=52&length=9)

Do you still use?

Reply:
Needd for the calls since the methods are part of hte HashState trait